opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -249,13 +252,51 @@ namespace Opm {
     }
 
 
+    template<typename FluidSystem, typename BlackoilIndices, typename ElementContext, typename MaterialLaw>
+    template <typename Simulator>
+    void
+    StandardWellsDense<FluidSystem, BlackoilIndices, ElementContext, MaterialLaw>::
+    getMobility(const Simulator& ebosSimulator, const int perf, const int cell_idx, std::vector<EvalWell>& mob) const
+    {
+
+        const int np = wells().number_of_phases;
+        assert (mob.size() == np);
+        const auto& intQuants = *(ebosSimulator.model().cachedIntensiveQuantities(cell_idx, /*timeIdx=*/0));
```

Comment:
This should not be  a problem for now as our wells are fully contained in the interior partition of the grid part. That my change sometime, of course.